code/modules/mob/living/carbon/inventory.dm

```diff
@@ -174,6 +174,33 @@
 
 	update_equipment_speed_mods()
 
+// Returns TRUE if an air tank compatible helmet is equipped.
+/mob/living/carbon/proc/can_breathe_helmet()
+	return head && isclothing(head) && (head.clothing_flags & MASKINTERNALS) ? TRUE : FALSE
+// Returns TRUE if an air tank compatible mask is equipped.
+/mob/living/carbon/proc/can_breathe_mask()
+	return wear_mask && isclothing(wear_mask) && (wear_mask.clothing_flags & MASKINTERNALS) ? TRUE : FALSE
+// Returns TRUE if a breathing tube is equipped.
+/mob/living/carbon/proc/can_breathe_tube()
+	return getorganslot(ORGAN_SLOT_BREATHING_TUBE) ? TRUE : FALSE
+// Returns TRUE if an air tank compatible mask or breathing tube is equipped.
+/mob/living/carbon/proc/can_breathe_internals()
+	return can_breathe_tube() || can_breathe_mask() || can_breathe_helmet()
+// Returns TRUE if air tank is open and mob lacks apparatus, or if the tank moved away from the mob.
+/mob/living/carbon/proc/invalid_internals()
+	return internal && (internal.loc != src || !can_breathe_internals()) ? TRUE : FALSE
```

Comment:
```suggestion
/mob/living/carbon/proc/invalid_internals()
	return !!(internal && (internal.loc != src || !can_breathe_internals()))
```
that or just make it an if, less confusion that way

Reply:
Hi, I disagree with your change because the ternary is more widely used in the codebase, and for this same purpose. Before using it, I spoke to several people in Discord and made sure to do research on how widely used the ternary was for this purpose. Furthermore, I personally think the double inversion is a superfluous operation when used in this kind of context.

Reply:
I can find 2 uses of ternaries like this (in that they return true if the case is true, and false otherwise). I don't like this use because it makes the conditional slightly visually muddy.
I can understand wanting to be explicit, but if you're going to be explicit just use if.

It would also be acceptable to just straight up return the string of blocks. I'm aware of the typesafety stuff, but that's an issue over pretty much the whole codebase, and if someone seriously tries to pull values out of the return value I'm gonna hittem over the head with a rock. (It's also niche enough behavior that I doubt anyone who would do it would ever know they could do it)

Reply:
I found 69 different instances of such a ternary in the TG codebase... but ok!

Reply:
Fixed by commit 3d3fc239644f88ab70d29e685536e8afd34f316d